### PR TITLE
Threads experimental transition modal

### PR DESCRIPTION
**Pull request by gsouquet**
Fixes https://github.com/vector-im/element-web/issues/21349

Will be displayed once, 1 minute after the user has loaded Element 

Screen Shot 2022-03-16 at 11 27 20

---
This PR currently has no changelog labels, so will not be included in changelogs.

Add one of: `T-Deprecation`, `T-Enhancement`, `T-Defect`, `T-Task` to indicate what type of change this is plus `X-Breaking-Change` if it's a breaking change.

Preview: https://pr8065--matrix-react-sdk.netlify.app
⚠️ Do you trust the author of this PR? Maybe this build will steal your keys or give you malware. Exercise caution. Use test accounts.

### CODE REVIEW

File: src/i18n/strings/en_EN.json [hunk 1]
```diff
@@ -3089,6 +3089,10 @@
...
+    "We’ve recently introduced key stability\n                                    improvements for Threads, which also means\n                                    phasing out support for experimental Threads.": "We’ve recently introduced key stability\n                                    improvements for Threads, which also means\n                                    phasing out support for experimental Threads.",
+    "All thread events created during the\n                                    experimental period will now be rendered in\n                                    the room timeline and displayed as replies.\n                                    This is a one-off transition. Threads are now\n                                    part of the Matrix specification.": "All thread events created during the\n                                    experimental period will now be rendered in\n                                    the room timeline and displayed as replies.\n                                    This is a one-off transition. Threads are now\n                                    part of the Matrix specification.",
```
Comment (luixxiul) [inline src/i18n/strings/en_EN.json:3094]: Are these `\n`s expected?

**Review by janogarcia — APPROVED** [review]

**Review by andybalaam — APPROVED** [review]
Looks good!

File: src/components/structures/MatrixChat.tsx [hunk 2]
```diff
@@ -1468,16 +1468,16 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
                             title: _t("Threads are no longer experimental! 🎉"),
                             description: <>
                                 <p>
-                                    { _t(`We’ve recently introduced key stability
-                                    improvements for Threads, which also means
-                                    phasing out support for experimental Threads.`) }
+                                    { _t("We’ve recently introduced key stability"
```
[the remark this answers is not in the document]
Comment (luixxiul) [reply src/components/structures/MatrixChat.tsx:1471]: sorry for being nit-picky!

File: src/components/structures/MatrixChat.tsx [hunk 3]
```diff
@@ -1460,6 +1461,36 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
...
+                                <p>
+                                    { _t("We’ve recently introduced key stability "
+                                    + "improvements for Threads, which also means "
+                                    + "phasing out support for experimental Threads.") }
+                                </p>
+                                <p>
+                                    { _t("All thread events created during the "
+                                    + "experimental period will now be rendered in "
+                                    + "the room timeline and displayed as replies. "
+                                    + "This is a one-off transition. Threads are now "
```
Comment (luixxiul) [inline src/components/structures/MatrixChat.tsx:1479]: While translating on Weblate I noticed that this line did not really match the one on the image inside the PR description. According to the image, this line should be `"This is one-off transition, and threads are now "`. Is this fine?

Comment (gsouquet) [reply src/components/structures/MatrixChat.tsx:1479]: The screenshot was out of date, we iterated on the text a bit

Comment (luixxiul) [reply src/components/structures/MatrixChat.tsx:1479]: OK I see, thanks for checking!

**Pull request merged**